isotovideo

```
@@ -410,16 +409,17 @@ if ($testfd) {
     kill_autotest;
 }
 
-diag "isotovideo done" unless $r;
-diag "FAIL" if $r;
+diag "isotovideo " . ($r ? 'failed' : 'done');
 
 my $clean_shutdown;
-eval {
-    $clean_shutdown = $bmwqemu::backend->_send_json({cmd => 'is_shutdown'});
-    diag "BACKEND SHUTDOWN $clean_shutdown";
-};
-
-bmwqemu::stop_vm();
+if (!$r) {
+    eval {
+        $clean_shutdown = $bmwqemu::backend->_send_json({cmd => 'is_shutdown'});
+        diag "BACKEND SHUTDOWN $clean_shutdown";
+    };
```

The exception is never handled but should be, at least in a log output